### ISSUE TITLE
Did we accidentally break top-level functions like get_table?

### ISSUE THREAD

**Issue by danielballan** [bluesky/databroker#212]
Note to self to follow up on this -- do the old top-level functions like `get_table`, `get_events`, and `get_images` still work?

They are definitely *not recommended* and should issue warnings (because they presume a singleton Broker instance) but I think we have to support them for awhile to avoid breaking many user notebooks.

If we did break these recently, I think we should reinstate them for the upcoming cycle -- again, with warnings.

**Comment by tacaswell** [comment]
I think we should break them on purpose.

**Comment by danielballan** [comment]
In in-person discussion, we agreed that we would continue to support old usages like:

```python
from databroker import get_table, db
get_table(db[-1])
```

with warnings that they will be removed in the future [because they are incompatible with using _multiple_ Brokers in the same process]. The recommended usage going forward will be documented in the upcoming release notes but I'll put it here too:

```python
# setup
from databroker import Broker
db = Broker.named('csx')  # refers to a config file at /etc/databroker/csx.yml

# usage
db[-1].table()
```

The old usage is now thoroughly protected by tests (merged in #220), so if/when we do finally remove it, it will be intentional. attn @cmazzoli, @bisogni, @yugangzhang

**Comment by cmazzoli** [comment]
Thanks.
Claudio
